### ISSUE TITLE
Lights badge counter misses lights

### ISSUE THREAD

**Issue by wrosenauer** [openhab/openhab-core#2342]
I'm running 3.1.0.M2 and found an issue with the badge counter on the location UI.
Not all of my lights are counted in some cases. Not exactly sure which combination makes it fail.
My report including screenshot of my model is here:
https://community.openhab.org/t/oh3-number-of-lights-turned-on-badge/121838

**Comment by cweitkamp** [comment]
Thanks for reporting.

Isn't this an UI issue and should go to https://github.com/openhab/openhab-webui/?

**Comment by wrosenauer** [comment]
> Isn't this an UI issue and should go to https://github.com/openhab/openhab-webui/?

You are the experts but you might be right. Can this be moved or does it need to be copied, if so?

**Comment by cweitkamp** [comment]
Yes, it can be moved. But I do not have the permissions to do that.
